app/bundles/CoreBundle/Test/AbstractMauticTestCase.php

```diff
@@ -96,21 +98,33 @@ protected function setUpSymfony(array $defaultConfigOptions = []): void
         EnvLoader::load();
 
         self::ensureKernelShutdown();
-        $this->client = static::createClient($this->clientOptions, $this->clientServer);
+        $this->client = static::createClient($this->clientOptions, $this->authenticateApi ? $this->clientServer : []);
         $this->client->disableReboot();
         $this->client->followRedirects(true);
 
         $this->em = static::getContainer()->get('doctrine')->getManager();
         \assert($this->em instanceof EntityManagerInterface);
         $this->connection = $this->em->getConnection();
-
-        $this->router = static::getContainer()->get('router');
-        $scheme       = $this->router->getContext()->getScheme();
-        $secure       = 0 === strcasecmp($scheme, 'https');
+        $this->router     = static::getContainer()->get('router');
+        $scheme           = $this->router->getContext()->getScheme();
+        $secure           = 0 === strcasecmp($scheme, 'https');
```

Comment:
```suggestion
        $this->router = static::getContainer()->get('router');
        $scheme       = $this->router->getContext()->getScheme();
        $secure       = 0 === strcasecmp($scheme, 'https');
```

Reply:
Is the suggested change just about indentation or am I missing something? It would break the CS Fixer rules.